### PR TITLE
feat: endpoint Dna Collection Process category and testing

### PR DESCRIPTION
**Pull request by JackyRahman**
### JIRA LINK
- https://blocksphere2020.atlassian.net/browse/DBIO-2588

### Changelog
- [feat: endpoint for dna collection process list](https://github.com/debionetwork/debio-backend/commit/77e871e735420ae423a13b04974ec3ed972275b0)
- [test: integration test dna collection process e2e](https://github.com/debionetwork/debio-backend/commit/cbb5de4b4c4c503b50fa7bd38f757f89e30ac98b)

### CODE REVIEW

File: src/endpoints/category/dna-collection/dna-collection.controller.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  @ApiResponse({
+    status: 200,
+    schema: {
+      example: dnaCollectionCategories,
+    },
+  })
+  getDnaCollectionCategory() {
+    return this.dnaCollectionService.getAll();
+  }
+}
```
Comment (ciniiia) [inline src/endpoints/category/dna-collection/dna-collection.controller.ts:25]: lgtm

**Review by ciniiia — APPROVED** [review]

File: src/endpoints/email/email.controller.ts [hunk 2]
```diff
@@ -68,6 +71,21 @@ export class EmailEndpointController {
...
+      description: `You've successfully submitted your account verification.`,
+      read: false,
+      created_at: await new Date(),
+      updated_at: await new Date(),
+      deleted_at: null,
+      from: 'Debio Network',
+      to: lab_id,
+    };
+
+    await this.notificationService.insert(notificationRegistration);
```
Comment (agustinustheo) [inline src/endpoints/email/email.controller.ts:87]: Notification services shouldn't exist within endpoints.

**Review by agustinustheo — CHANGES_REQUESTED** [review]

**Comment by agustinustheo** [comment]
Also please fix the conflicts, and follow the guidelines from here: https://github.com/debionetwork/debio-backend/pull/389

File: src/endpoints/email/email.controller.ts (hunk 2 above)
Comment (JackyRahman) [reply src/endpoints/email/email.controller.ts:87]: updated

**Review by agustinustheo — APPROVED** [review]

**Review by agustinustheo — APPROVED** [review]

**Pull request merged**